lavalink/lavalink.go

```diff
@@ -249,8 +249,8 @@ func (l *lavalinkImpl) Close() {
 }
 
 func (l *lavalinkImpl) VoiceServerUpdate(voiceServerUpdate VoiceServerUpdate) {
-	player := l.players[voiceServerUpdate.GuildID]
-	if player == nil && player.LastSessionID() != nil {
```

Comment:
this check needs to be here in order to prevent a NPE thrown in the `Node.Send()`

the session is required for lavalink to connect afaik <https://github.com/freyacodes/Lavalink/blob/dev/IMPLEMENTATION.md#provide-a-voice-server-update>

I don't see why we should send a voice update without it